src/store/modules/collections.js

```diff
@@ -91,12 +91,7 @@ export default {
       { collection_id, data }
     ) {
       if (validateID(collection_id) && validateParams(data)) {
-        const material = await this.$axios.$post(
-          `collections/${collection_id}/materials/`,
-          data
-        );
-        commit('SET_MATERIAL_TO_COLLECTION', material);
```

Comment:
So this thing was killing us. The rest I removed because it doesn't seem to make sense to return your input directly.